spec/controllers/revision_feedback_controller_spec.rb

```diff
@@ -3,28 +3,34 @@
 
 describe RevisionFeedbackController do
   describe '#index' do
-    let(:params) { { rev_id: 123456 } }
-    let(:revision) { build(:revision, mw_rev_id: 123456) }
+    let(:params) { { article_id: 1 } }
+    let(:article) { build(:article, { mw_page_id: 27697087, id: 1 }) }
 
-    context 'when the revision is in the database' do
+    context 'When the article exists' do
       before do
-        revision.save
-        get :index, params: params
+        VCR.use_cassette 'ores_features' do
+          article.save
+          get :index, params: params
+        end
       end
 
       it 'renders without error' do
         expect(response.status).to eq(200)
       end
-    end
 
-    context 'when the revision is not in the database' do
-      it 'imports and saves the revisions' do
-        expect(Revision.count).to eq(0)
-        expect(Revision).to receive(:new).and_return(revision)
-        expect_any_instance_of(RevisionScoreImporter).to receive(:update_revision_scores)
-        get :index, params: params
-        expect(response.status).to eq(200)
-        expect(Revision.count).to eq(1)
+      it 'calls RevisionFeedbackService with features' do
+        VCR.use_cassette 'ores_features' do
+          expect_any_instance_of(RevisionFeedbackService).to receive(:initialize)
+            .with(have_key('feature.enwiki.revision.cite_templates'))
+          expect_any_instance_of(RevisionFeedbackService).to receive(:feedback)
+            .and_return(have_at_least(1))
+          get :index, params: params
+        end
+      end
+
+      it 'assigns feedback properly' do
+        feedback = assigns(:feedback)
+        expect(feedback.length).to be >= 1
```

Comment:
These specs could use some comments.

It took me a while to realize that these tests — I think — don't depend on the the state of the latest revision of article 27697087. (They just depend on it existing and ORES returning the normal features hash for it.)

Adding what the article is, and explaining in a comment how this is just acting as an integration test with `RevisionFeedbackController` and `RevisionFeedbackService` — but without testing the particular revision-dependent results of either one — would be helpful.